src/ConstructiveSolidGeometry/LinePrimitives/Arc.jl

```diff
@@ -0,0 +1,58 @@
+struct Arc{T,TP,TH} <: AbstractLinePrimitive{T}
+    r::T
+    center::TP
```

Comment:
As `center` will always be `PlanarPoint{T}`, ditch `TP` and write `center::PlanarPoint{T}`.

Reply:
The idea was that`Arc` could also be a 3D object like line. But I did not add that option